modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -104,91 +104,12 @@ The following command applies to a single cluster named `du-sno-1-zone-1` and th
-
-    - fileName: PerformanceProfile.yaml # wave 10
-      policyName: "group-du-sno-cfg-policy"
-      metadata:
-        name: openshift-node-performance-profile
-      spec:
-        additionalKernelArgs:
-        - rcupdate.rcu_normal_after_boot=0
-        - vfio_pci.enable_sriov=1
-        - vfio_pci.disable_idle_d3=1
-        - efi=runtime
-        cpu:
-          isolated: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-isolated" (index .ManagedClusterLabels "hardware-type")) hub}}'
-          reserved: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-reserved" (index .ManagedClusterLabels "hardware-type")) hub}}'
-        hugepages:
-          defaultHugepagesSize: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-default" (index .ManagedClusterLabels "hardware-type")) hub}}'
-          pages:
-            - size: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-size" (index .ManagedClusterLabels "hardware-type")) hub}}'
-              count: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-hugepages-count" (index .ManagedClusterLabels "hardware-type")) | toInt hub}}'
-        realTimeKernel:
-          enabled: true
-
-    - fileName: SriovNetwork.yaml # wave 100
-      policyName: "group-du-sno-sriov-policy"
-      metadata:
-        name: sriov-nw-du-fh
-      spec:
-        resourceName: du_fh
-        vlan: '{{hub fromConfigMap "" "site-data-configmap" (printf "%s-sriov-network-vlan-1" .ManagedClusterName) | toInt hub}}'
-        
-    - fileName: SriovNetworkNodePolicy.yaml # wave 100
-      policyName: "group-du-sno-sriov-policy"
-      metadata:
-        name: sriov-nnp-du-fh
-      spec:
-        deviceType: netdevice
-        isRdma: false
-        nicSelector:
-          pfNames: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-sriov-node-policy-pfNames-1" (index .ManagedClusterLabels "hardware-type")) | toLiteral hub}}'
-        numVfs: 8
-        priority: 10
-        resourceName: du_fh
-
-    - fileName: SriovNetwork.yaml # wave 100
-      policyName: "group-du-sno-sriov-policy"
-      metadata:
-        name: sriov-nw-du-mh
-      spec:
-        resourceName: du_mh
-        vlan: '{{hub fromConfigMap "" "site-data-configmap" (printf "%s-sriov-network-vlan-2" .ManagedClusterName) | toInt hub}}'
-
-    - fileName: SriovNetworkNodePolicy.yaml # wave 100
-      policyName: "group-du-sno-sriov-policy"
-      metadata:
-        name: sriov-nw-du-fh
-      spec:
-        deviceType: netdevice
-        isRdma: false
-        nicSelector:
-          pfNames: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-sriov-node-policy-pfNames-2" (index .ManagedClusterLabels "hardware-type")) | toLiteral hub}}'
-        numVfs: 8
-        priority: 10
-        resourceName: du_fh
+include::snippets/pg-ztp-specifying-nics-in-pgt-hub-cluster-templates.yaml[]
```

Comment:
In files updated with PolicyGenerator, I would keep the PGT examples but add a new snippet with the code for PolicyGenerator. Since PGT is not deprecated yet, we should not remove it.

Reply:
I can't do that easily, but what I've done is group all the PGT stuff together under a new heading for PGT
![image](https://github.com/openshift/openshift-docs/assets/74046732/c43d5c95-23d2-4645-90fb-7210be2a277f)

let me know if this works for you. The PGT content is all still there, just moved together under the "Using PolicyGenTemplate resources with GitOps ZTP" section in the TOC.